chrome-app/links.js

```diff
@@ -17,10 +17,13 @@ const ourRepos = ["repo:aws/jsii", "repo:aws/aws-cdk"];
 
 // LINKS
 const LINKS = [
-    { title: "Review", href: "https://github.com/pulls/review-requested" },
     { title: "Triage", href: gitHubQuery("https://github.com/issues", [...issue, ...ourRepos, "assignee:USERNAME", "label:bug", "-label:p0", "-label:p1", "-label:p2", "-label:p3"])},
+    { title: "P0", href: gitHubQuery("https://github.com/issues", [...issue, ...ourRepos, "assignee:USERNAME", "label:bug", 'label:p0'])},
     { title: "Shepherd", href: gitHubQuery("https://github.com/pulls", [...pr, "assignee:USERNAME", "-author:USERNAME" ])},
-    { title: "Finish", href: "https://github.com/pulls" },
-    { title: "Your bugs", href: gitHubQuery("https://github.com/issues", [...issue, ...ourRepos, "assignee:USERNAME", "label:bug"])},
-    { title: "Your issues", href: "https://github.com/issues/assigned" },
+    { title: "Author", href: "https://github.com/pulls" },
+    { title: "Review", href: "https://github.com/pulls/review-requested" },
+    { title: "P1", href: gitHubQuery("https://github.com/issues", [...issue, ...ourRepos, "assignee:USERNAME", "label:bug", 'label:p1'])},
```

Comment:
All the P-everythings feel too fine-grained. Can we do this in fewer categories?

Reply:
I think we need a "P0" category for sure. So maybe we can do "P0" and "Other Bugs" (that's the only viable option given you can't do "OR" conditions in queries)

Reply:
Fixed